Sources/Datadog/RUM/RUMTelemetry.swift

```diff
@@ -81,18 +97,21 @@ internal final class RUMTelemetry: Telemetry {
     /// and action ID.
     ///
     /// - Parameters:
+    ///   - id: Identity of the debug log, this can be used to prevent duplicates.
     ///   - message: Body of the log
     ///   - kind: The error type or kind (or code in some cases).
     ///   - stack: The stack trace or the complementary information about the error.
-    func error(_ message: String, kind: String?, stack: String?) {
+    func error(id: String, message: String, kind: String?, stack: String?) {
         guard
+            accept(id: id),
             let monitor = Global.rum as? RUMMonitor,
             let writer = RUMFeature.instance?.storage.writer
         else {
             return
         }
 
         let date = dateCorrector.currentCorrection.applying(to: dateProvider.currentDate())
+        events.append(id)
```

Comment:
If we store event IDs, wouldn't it be better to call it `eventIDs` instead? Alternatively, the `event's` comment could explain the actual content.